source/software/matlab_parallel_computing.rst

```diff
@@ -3,12 +3,155 @@
+
+> **_NOTE:_**  Be careful: when leaving this field blank, this location will default to your ``$VSC_HOME``, which could result in quickly filling up your home directory.
+
+Next, you'll need to load your preferred module, and start it. It is recommended to do this via an interactive job.
+
+::
+
+   $ # Genius
+   $ module use /apps/leuven/skylake/2021a/modules/all
+   $ module load MATLAB/R2022b #MATLAB/R2022a if you would like to load this version
+    
+Once the module is loaded, start Matlab. You can start it in a terminal without the GUI by using the ``-nodisplay`` flag. Then, run the ``configCluster`` script.
+This will set up the parallel configuration.  
+
+::
+
+   $ matlab -nodisplay
+
+::
+
+   >> rehash toolbox;
+   >> configCluster;
+   
+You will get a message as follows if it succeeded: 'Complete.  Default cluster profile set to "<cluster_name> <matlab_version>".'. You have now created a cluster
+profile.
+
+Configuring additional properties
+*********************************
+
+It is possible to set some user-specific properties and save them in your cluster profile. You can always adapt these and save the profile again, without performing
+the previous step again. All of these properties are available in the ``AdditionalProperties´´ property of your cluster profile. 
+   
+Before you will be able to adapt these properties, you first need to get a handle on the cluster:
+
+::
+
+   >> c = parcluster;
+   
+``c = parcluster´´ will automatically load the correct profile based on the Matlab module you have loaded. So if you would have a profile for both R2022a and R2022b,
+the correct profile will be chosen when using this command.
+
+If you would prefer to use the local resources (i.e. the resources of the node you are currently on), use:
+
+::
+
+   >> c = parcluster('local');
+
+You can set a range of additional properties on each profile. There are two properties that are required, namely ``AccountName`` and ``WallTime``. Set any 
+other to your preference. Once these are saved, they will be kept in the settings of your cluster profile, but they can always be adapted again later.
+
+::
+
+   >> % View the additional properties
+   >> c.AdditionalProperties
+   >> % set the required properties
+   >> c.AdditionalProperties.AccountName = <account-name>;
+   >> c.AdditionalProperties.WallTime = '05:00:00';
+   >> % save the profile
+   >> c.saveProfile
+   
+Now the profile is ready, and you should be able to submit multi-node jobs with Matlab now.
+
+Submitting jobs
++++++++++++++++
+
+Submitting jobs will happen from within Matlab. First of all, you will need to start an interactive session on the cluster. It is possible to use the GUI, but then
+you will need to request GPU resources. You can also open the GUI on NX, but you need to submit your jobs from a compute node. Submitting from a login node
+will cause an error. 
```

Comment:
I think this also needs some more polishing. In particular the sentence ``You can also open the GUI on NX, but you need to submit your jobs from a compute node.`` seems contradictory. Maybe it's best to just first describe the different possible for the GUI approach, and after that describing the CLI approach (and making clear in both cases on which kind of nodes one's Matlab prompt is supposed to be running for it to work).

Reply:
Yes, I was not sure about how much detail I need to add here. I assume that most Matlab users are aware that they can open the GUI on NX, but like I mentioned, you cannot submit multi-node jobsfrom there. So if you want to be able to submit multi-node jobs AND use a GUI, you should request GPU resources. But, I'm not really keen on giving too much detail on that, as I do not want to push users to out GPU resources if it is just for running a GUI.

Reply:
Yes, then you can indeed put most emphasis on the CLI approach. If you don't want to describe the full situation for the GUI, then you can focus on the specific case of using-the-GUI-inside-a-NX-session and then add a note that this will not allow for multi-node jobs.